### PR TITLE
frontend/feat/driver cancellation rate rolling window

### PR DESCRIPTION
**Pull request by shreyaxh**
e:   Frontend/android-native/app/src/driver/common/res/drawable/ny_ic_gauge_tip.png

## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added integration tests for my changes where possible
- [ ] I tested the changes using integration tests
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Frontend/ui-driver/src/Screens/CancellationRateScreen/View.purs [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+          , height $ V 100
+          , gravity LEFT
+          ]
+        , linearLayout
+          [ gravity CENTER
+          , width MATCH_PARENT
+          ][
+            imageView
+            [ imageWithFallback $ fetchImage FF_ASSET "ny_ic_gauge_tip"
+            , width $ V $ round (((toNumber (screenWidth unit - 32)/ 2.0) ) * (11.0 / 75.0))
```
Comment (prakharritik) [inline Frontend/ui-driver/src/Screens/CancellationRateScreen/View.purs:169]: What is logic for 11/75

File: Frontend/ui-driver/src/Screens/DriverProfileScreen/ScreenData.purs [hunk 2]
```diff
@@ -92,7 +92,13 @@ initData =
       , totalRidesAssigned : 0
       , totalDistanceTravelled : ""
       },
-    config
+    config,
+    cancellationRate : 30,
+    assignedRides : 100,
+    cancelledRides : 69,
```
[the remark this answers is not in the document]
Comment (shreyaxh) [reply Frontend/ui-driver/src/Screens/DriverProfileScreen/ScreenData.purs:98]: no

File: Frontend/ui-driver/src/Screens/DriverProfileScreen/View.purs [hunk 3]
```diff
@@ -887,6 +899,104 @@ addRcView state push =
...
+            , height $ V 100
+            , gravity LEFT
+            ]
+          , linearLayout
+            [ gravity CENTER
+            , width MATCH_PARENT
+            ][
+              imageView
+              [ imageWithFallback $ fetchImage FF_ASSET "ny_ic_gauge_tip"
+              , width $ V $ round (((toNumber (screenWidth unit - 32)/ 2.0) ) * (11.0 / 75.0))
```
Comment (prakharritik) [inline Frontend/ui-driver/src/Screens/DriverProfileScreen/View.purs:941]: Can you make a common view for this?

Comment (shreyaxh) [reply Frontend/ui-driver/src/Screens/DriverProfileScreen/View.purs:941]: no

File: Frontend/ui-driver/src/Screens/HomeScreen/View.purs [hunk 4]
```diff
@@ -1054,6 +1057,18 @@ driverProfile push state =
           , height $ V 42
           , imageWithFallback $ HU.fetchImage HU.FF_ASSET driverImage
           ]
+        , imageView
+          [ width $ V 42
+          , height $ V 42
+          , imageWithFallback $ HU.fetchImage HU.FF_ASSET "ny_ic_orange_pfp_ring"
```
Comment (prakharritik) [inline Frontend/ui-driver/src/Screens/HomeScreen/View.purs:1063]: here we can make one view and add check for image name

File: Frontend/ui-driver/src/RemoteConfigs/Types.purs [hunk 5]
```diff
@@ -32,4 +36,25 @@ type HVConfigs = {
...
   aadhaar_flow_id :: String
+}
+
+data CancellationRateEntity = HOW_TO_REDUCE | WHAT_HAPPENS
+
+derive instance genericCancellationRateEntity :: Generic CancellationRateEntity _
+instance decodeCancellationRateEntity :: Decode CancellationRateEntity where decode = defaultEnumDecode
+instance encodeCancellationRateEntity :: Encode CancellationRateEntity where encode = defaultEnumEncode
+instance eqCancellationRateEntity :: Eq CancellationRateEntity where eq = genericEq
+instance showCancellationRateEntity :: Show CancellationRateEntity where
```
Comment (prakharritik) [inline Frontend/ui-driver/src/RemoteConfigs/Types.purs:47]: Can you check if this is required?

File: Frontend/ui-driver/src/Screens/DriverProfileScreen/View.purs [hunk 6]
```diff
@@ -887,6 +899,104 @@ addRcView state push =
...
+          , orientation VERTICAL
+          , gravity CENTER
+          ] [ 
+              textView $
+              [ width MATCH_PARENT
+              , height WRAP_CONTENT
+              , text $ show cancellationRate <> "%"
+              , color Color.black900
+              , padding $ PaddingBottom 4
+              , textSize FontSize.a_24
```
Comment (prakharritik) [inline Frontend/ui-driver/src/Screens/DriverProfileScreen/View.purs:960]: textSize can be removed

File: Frontend/ui-driver/src/Screens/DriverProfileScreen/View.purs [hunk 7]
```diff
@@ -887,6 +899,104 @@ addRcView state push =
...
+              , text (getString CANCELLATION_RATE)
+              , color Color.black700
+              , textSize FontSize.a_14
+              , padding $ Padding 11 0 11 8
+              , gravity CENTER
+              ] <> FontStyle.body3 LanguageStyle
+            , textView $
+              [ width MATCH_PARENT
+              , height WRAP_CONTENT
+              , text "View more"
```
Comment (prakharritik) [inline Frontend/ui-driver/src/Screens/DriverProfileScreen/View.purs:975]: Add translation

**Pull request closed without merging**